client/me/purchases/purchases-list-in-dataviews/purchases-data-change.tsx

```diff
@@ -0,0 +1 @@
+// This is where onChangeView will eventually be implemented
```

Comment:
I would avoid adding this empty file for now until you need it.

Reply:
Done